classes/class-p4-search.php

```diff
@@ -27,12 +24,6 @@ abstract class P4_Search {
 		const DEFAULT_CACHE_TTL     = 600;
 		const DUMMY_THUMBNAIL       = '/images/dummy-thumbnail.png';
 		const EXCLUDE_FROM_SEARCH   = 'p4_do_not_index';
-		const POST_TYPES            = [
```

Comment:
Replaced this with https://github.com/greenpeace/planet4-master-theme/pull/1039/files#diff-8332426ee2026a248f5fc7cf275911c3R515